src/main/java/com/github/markusbernhardt/seleniumlibrary/locators/TableElementFinder.java

```diff
@@ -50,6 +50,7 @@ public static WebElement findByHeader(WebDriver webDriver, String tableLocator,
 
 	public static WebElement findByFooter(WebDriver webDriver, String tableLocator, String content) {
 		List<String> locators = parseTableLocator(tableLocator, "footer");
+		System.out.println(Arrays.toString(locators.toArray()));
```

Comment:
System.out.println(String.format("Find by footer locators: %s", locators));

Reply:
Does this always output the message to a console? As a library user I don't want to see debug messages unless I'm in a debug mode.

Reply:
I agree with you but I think if don’t remove the output, it’s better to make it readable for any users))

Reply:
This was just for debugging, used System.out.println to easily spot the ones needed to be removed.

Reply:
ok. I think it can be removed from PR)